integrations/mongodb_atlas/src/haystack_integrations/document_stores/mongodb_atlas/document_store.py

```diff
@@ -19,6 +19,39 @@
 
 
 class MongoDBAtlasDocumentStore:
+    """
+    MongoDBAtlasDocumentStore is a DocumentStore implementation that uses MongoDB Atlas - a fully managed cloud database
```

Comment:
no need to make advertisement, no? 😁 
In any case I'd add a link so users can explore on their own
```suggestion
    MongoDBAtlasDocumentStore is a DocumentStore implementation that uses [MongoDB Atlas](https://www.mongodb.com/atlas/database).
```